6_transformer_translation.ipynb

```diff
@@ -0,0 +1,1266 @@
+{
```

Comment:
<p><code>.9999</code> how many lines of data are there??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scoutbee/pytorch-nlp-notebooks/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
145437 in total, I just want a few lines of prediction output in test_loader